kong/runloop/plugin_servers/init.lua

```diff
@@ -310,16 +320,15 @@ end
 
 
 function plugin_servers.start()
-  if ngx.worker.id() ~= 0 then
-    kong.log.notice("only worker #0 can manage")
-    return
-  end
-
   local pluginserver_timer = proc_mgmt.pluginserver_timer
+  local pluginserver_connection_check_timer = proc_mgmt.connection_check_timer
 
   for _, server_def in ipairs(proc_mgmt.get_server_defs()) do
     if server_def.start_command then
-      _G.native_timer_at(0, pluginserver_timer, server_def)
+      if ngx.worker.id() == 0 then
+        _G.native_timer_at(0, pluginserver_timer, server_def)
```

Comment:
We can use `local` to cache `_G.native_timer_at` in the for-loop

Reply:
The problem, still, is that we cannot be sure if the variable is ready when we load the module.
So the best we can do is to cache the function in the local scope(which is not very meaningful)